plugins/pagerduty/src/components/Incident/IncidentListItem.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  ListItem,
+  ListItemIcon,
+  ListItemSecondaryAction,
+  Tooltip,
+  ListItemText,
+  makeStyles,
+  IconButton,
+  Link,
+  Typography,
+} from '@material-ui/core';
+import { StatusError, StatusWarning } from '@backstage/core';
+import moment from 'moment';
```

Comment:
Just FYI: There's an effort going on to replace the (discontinued) moment library with alternatives.

Reply:
https://github.com/backstage/backstage/issues/3401. Please add comments if you have any opinions or thoughts :)